react/__tests__/OrderBy.test.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-env jest */
+import React from 'react'
+import { render } from 'test-utils'
+
+import OrderBy from '../OrderBy'
+import { setMobileState } from 'vtex.render-runtime'
+
+describe('<OrderBy />', () => {
+  const renderComponent = customProps => mobile => {
+    setMobileState(mobile)
+    const props = {
+      getLinkProps: () => ({ page: 'mokedPage' }),
+      orderBy: 'OrderByTopSaleDESC',
+      ...customProps,
+    }
+
+    return render(<OrderBy {...props} />)
+  }
+
+  it('should match snapshot in web mod', () => {
+    const { asFragment } = renderComponent()(false)
```

Comment:
Also, don't you tIt hink that it would be better to set the mobile state in each test. It would be more readable what this true/false is doing.